src/components/Modal/ModalClose.vue

```diff
@@ -0,0 +1,17 @@
+<script setup>
+</script>
+
+<template>
+   
+        <div class="modal-default-button" @click="$emit('close')">
```

Comment:
wierd name of the class - a main component`s class usually linked to the component name

Reply:
rename to modal__button